src/handler.py

```diff
@@ -192,8 +194,10 @@ def parse_and_dispatch(var,
                 wrapper.pm(messages["no_force_admin"])
                 return
             if fn.chan:
+                assert channels.Main is not None
```

Comment:
this is never None in the course of normal bot operation (with exception of early bootstrapping), so it should be typed as such rather than adding spurious asserts elsewhere. Ditto users.Bot